src/components/app-configure/AppUserLimits.vue

```diff
@@ -64,6 +63,12 @@
               :disabled="storageUnlimited"
             />
           </div>
+          <span
+            class="error-message"
+            :class="storageError ? { show: true } : ''"
+          >
```

Comment:
Instead of using the `error-message.show` class to toggle visibility, can we use `v-if`?

```suggestion
          <span
            v-if="storageError"
            class="error-message"
          >
```

Reply:
Yeah initially I used v-if, but it renders component only when conditions are met, which is fine. I wanted the component to use that space while still being hidden hence I used styles like `visibility: hidden` and in show I used `visibility: visible`. Maybe we can use `v-show` as well. Let me know

Reply:
Ah, I see. The `v-show` directive toggles `display` which won't help in this case. Let's continue with the `.show` class then. 

We can use the [object syntax](https://v3.vuejs.org/guide/class-and-style.html#object-syntax) for the class binding.

```suggestion
          <span
            class="error-message"
            :class="{ show: hasStorageError }"
          >
```